docs/data/charts/line-demo/line-demo.md

```diff
@@ -31,3 +31,15 @@ components: LineChart, LineElement, LineHighlightElement, LineHighlightPlot, Lin
 ## LineChartConnectNulls
 
 {{"demo": "LineChartConnectNulls.js"}}
+
+## Line with predicition
```

Comment:
```suggestion
## Line with multiple styles
```
I don't think this is specific for prediction, what do you think about having the "prediction" keyword in the example but not on the title?

Reply:
The advantage of having it in the title is that it will be used by the Algolia search
So if we think people will search for this word, it's interesting
Not sure they will though

Reply:
Algolia also searches for the body, I've added `estimated` and `styles` to allow for broader terms

Reply:
> Algolia also searches for the body

With a similar importance?

Reply:
No, but we also can't have a section title with all the keywords like a news blog 😂 `Line with multiple styles for prediction and estimations`

Reply:
since multiple and style are already used a lot, I prefer avoiding adding noise on those key words

ABout the discoverability of the example, I assume if it's hard to find we will get issues. It will be the opportunity to rethink how we organise the docs.

At the begining those pages were here mostly to compare our render with the Recharts examples (the reason why most of the demos are with weird title)

Reply:
If we want to use market terms I would probably use `forecast` or `projection` instead of `prediction`? Though I'm not a specialist. Regardless, we at least have to fix the extra `i` in the current title.

Reply:
```suggestion
## Line with forecast
```